src/spryker/application/skeleton/composer-harness.json

```diff
@@ -30,9 +30,12 @@
       "@test-unit",
       "@test-acceptance"
     ],
-    "test-quality": [
+    "devtools-setup": [
```

Comment:
would `generate` be a better name?

Reply:
(similiar in terminology as `go generate`

Reply:
Updated :+1: